src/modules/dashboard/components/CoinMachine/RemainingDisplayWidget/RemainingDisplayWidget.tsx

```diff
@@ -217,9 +219,13 @@ const RemainingDisplayWidget = ({
           [styles.valueWarning]: isValueWarning || showValueWarning,
         })}
       >
-        {displayedValue}
+        {saleStarted ? (
+          displayedValue
+        ) : (
+          <FormattedMessage {...widgetText.placeholder} />
+        )}
       </p>
-      {periodTokens && widgetText.footerText && (
+      {saleStarted && periodTokens && widgetText.footerText && (
```

Comment:
we don't need the `saleStarted` check here either as it's already in `periodTokens` in CoinMachine. If the sale hasn't started it will return undefined and hence `periodTokens` will be converted to `false`.
This way (together with my previous comment) we can remove `saleStarted` prop from the whole component.